app.js

```diff
@@ -134,7 +134,9 @@ var shuffleColors = function() {
 }
 //when reset button is pressed to refresh page
 function refresh() {
-    location.reload();
+    document.getElementById("text").style.background = 'rgb(80, 151, 151)';
+    document.getElementById("background").style.background = 'lightgrey';
```

Comment:
got a bug in your randomizer for the images every once in a while one stays the same and or 2 of the same image. 
work on that and resubmit. looking good other than that.